DOCS-3950: Change default module version (#4312)
viamrobotics/docs: docs/operate/reference/module-configuration.md

#### Module versioning

You can configure how each module on your machine updates itself when a newer version becomes available from the Viam Registry.
By default, a newly-added module is set to pin to the latest release (**Latest**) of the version you added.

> To allow automatic module updates when a new version of the module becomes available in the Viam Registry, set the **Pinned version type** for your module in its module card on the **CONFIGURE** tab.

Author: ```suggestion
To change the update strategy for your module, set the **Pinned version type** for your module in its module card on the **CONFIGURE** tab.
```
